index.bs

```diff
@@ -463,21 +463,28 @@ given a [=request=] |request|, perform the following steps:
 
 1. Let |origin| be |request|'s [=request/origin=].
 1. If |origin| is an [=opaque origin=], then abort these steps.
-1. If |request|'s [=request/destination=] is "`document`", then:
+1. If |request| is a [=subresource request=], then:
```

Comment:
If we are handling subresource requests here then I don't think the algorithm should be named specific to "navigation".  Navigation implies non-subresources.

Reply:
Done.